FluidDial/src/JoggingScene.cpp

```diff
@@ -7,19 +7,19 @@
 class JoggingScene : public Scene {
 private:
     static const int MAX_INC = 5;
-    static const int n_axes  = 3;
+    //static const int n_axes  = 6;
```

Comment:
Just delete this instead of leaving in dead code.  Other than that, this PR is okay with me.

Reply:
done, thanks